app/scripts/lib/notification-manager.js

```diff
@@ -1,7 +1,10 @@
 const extension = require('extensionizer')
-const height = 620
-const width = 360
-
+const SCREEN_WIDTH = window.screen.width
+const SCREEN_HEIGHT = window.screen.height
```

Comment:
Since a screen's resolution can be adjusted at run-time, we should maybe re-compute this at popup creation time.

Reply:
Good idea! I'll fix this right up.